packages/rs-drive/src/drive/verify/identity.rs

```diff
@@ -563,16 +566,23 @@ impl Drive {
         } else {
             GroveDb::verify_query(proof, &path_query)?
         };
-        if proved_key_values.len() == public_key_hashes.len() {
-            let values = proved_key_values
-                .into_iter()
-                .map(|proved_key_value| {
-                    let key: [u8; 20] = proved_key_value
-                        .1
-                        .try_into()
-                        .map_err(|_| Error::Proof(ProofError::IncorrectValueSize("value size")))?;
-                    let maybe_element = proved_key_value.2;
-                    match maybe_element {
+        let mut proved_key_values_hm: HashMap<[u8; 20], PathKeyOptionalElementTrio> = HashMap::new();
```

Comment:
we  shouldn't do this change, it should be handled inside grovedb - it should return correct length

Reply:
reverted changes